web/src/components/storage/dasd/DASDTable.tsx

```diff
@@ -78,29 +80,58 @@ const columns = [
   { id: "formatted", label: _("Formatted") },
   { id: "partitionInfo", label: _("Partition Info") },
 ];
+const DevicesList = ({ devices }) => (
+  <ul>
+    {devices.map((d: DASDDevice) => (
+      <li key={d.id}>{d.id}</li>
+    ))}
+  </ul>
+);
+const FormatNotPossible = ({ devices, onAccept }) => (
+  <Popup isOpen title={_("Offline DASD devices")}>
+    <Text>
+      {_(
+        "The DASD devices listed below are offline and cannot be formatted, please unselect or activate them in order to continue",
+      )}
+    </Text>
```

Comment:
Same than the other comment, let's try to improve both, the title and the body if you don't mind.

Ideas, 

* Title: `Cannot format all selected devices`
* Body

   > Offline devices cannot be formatted. Please, unselect or activate below devices and try it again.
   > [devices]

Again, I suck at wording but as said, these are just examples of 

* More informative and understandable title
* More explanatory and clear body